src/ui.rs

```diff
@@ -350,6 +351,16 @@ impl Ui {
         self.set_total(total_hours);
 
         self.grid.show_all();
+
+        match scroll_bottom {
+            Some(scroll) => {
+                if scroll {
+                    let adjustment = self.scroll_view.get_vadjustment().unwrap();
+                    adjustment.set_value(adjustment.get_upper() - adjustment.get_page_size());
```

Comment:
It appears this code works from within a size adjustment event for grid. Something like this works:

```
self.grid.connect_size_allocate(clone!(scroll_view => move |_,_| {
  let adj = scroll_view.get_vadjustment().unwrap();
  adj.set_value(adj.get_upper() - adj.get_page_size());
}));
```
However, after adding this event handler, any other event will still scroll to bottom, resulting in an unusable edit of the first item for example. Any ideas?

Reply:
thats why i am only try to scroll if creating a new item without using this event handler.

dumb idea from my side: add such a handler on top of the "set_time_entries" method, if a new item was added (scroll_bottom param is true) and at the end we could try to remove this listener.

I will check it